vuu-ui/packages/vuu-shell/src/layout-management/LayoutList.css

```diff
@@ -1,21 +1,21 @@
+.vuuLayoutList {
+    align-self: stretch;
+}
+
```

Comment:
Could you explain the changes in this file, please?

Reply:
sure: align-self here works similarly to align-items inside a flex container. so we are telling the layoutList to stretch along the x axis to take all the space that is available

Reply:
this fixes the line at the bottom of the ListHeader: 
<img width="196" alt="1" src="https://github.com/ScottLogic/finos-vuu/assets/98337074/65c30c19-7f5d-4251-8779-b2443f382355">
<img width="195" alt="2" src="https://github.com/ScottLogic/finos-vuu/assets/98337074/258831e8-6784-4359-94a2-523c33d38390">